shell/browser/extensions/extension_tab_util.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SHELL_BROWSER_EXTENSIONS_EXTENSION_TAB_UTIL_H_
+#define SHELL_BROWSER_EXTENSIONS_EXTENSION_TAB_UTIL_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/callback.h"
+#include "extensions/common/features/feature.h"
+#include "shell/common/extensions/api/tabs.h"
+#include "ui/base/window_open_disposition.h"
+
+class GURL;
+class ExtensionFunction;
+
+namespace electron {
+namespace api {
+struct ExtensionTabDetails;
+class WebContents;
+}  // namespace api
+
+}  // namespace electron
+
+namespace base {
+class DictionaryValue;
+class ListValue;
+}  // namespace base
+
+namespace content {
+class BrowserContext;
+class WebContents;
+}  // namespace content
+
+namespace gfx {
+class Rect;
+}
```

Comment:
```suggestion
namespace electron {
namespace api {
struct ExtensionTabDetails;
class WebContents;
}  // namespace api

}  // namespace electron

namespace content {
class WebContents;
}  // namespace content

```

Appear to be unused in this file.